scripts/common.sh

```diff
@@ -40,6 +40,26 @@ else
     function printLog { echo "$(tput setaf 3)$1$(tput sgr0)"; }
 fi
 
+function checkDputEntries
+{
+    local pattern="$1"
+    grep "${pattern}" /etc/dput.cf --quiet || \
+        fail "Error: Missing ${pattern//\\/} section in /etc/dput.cf (check top comment in ${0} for more information)."
```

Comment:
I guess not every file where we might use this will have this comment.
```suggestion
        fail "Error: Missing ${pattern//\\/} section in /etc/dput.cf (check top comment in release_ppa.sh for more information)."
```

Or maybe it would be better to have something like `common_ppa.sh` and put the comments there?

Reply:
Same for `sourcePPAConfig()`.